common/src/main/java/org/opensearch/sql/common/setting/Settings.java

```diff
@@ -29,6 +29,7 @@ public enum Key {
     PATTERN_MODE("plugins.ppl.pattern.mode"),
     PATTERN_MAX_SAMPLE_COUNT("plugins.ppl.pattern.max.sample.count"),
     PATTERN_BUFFER_LIMIT("plugins.ppl.pattern.buffer.limit"),
+    SPL_COMPATIBLE_GRAMMAR_ENABLED("plugins.ppl.spl_compatible.enabled"),
```

Comment:
any other options? such as `plugins.ppl.splunk_compatible.enabled` to highlight `splunk` instead of `spl`